lib/app.js

```diff
@@ -5,6 +5,7 @@ const express = require('express');
 const config = require('./config');
 const SioFb = require('./sioFb');
 const SioBot = require('./sioBot');
+const pkg = require('./../package.json');
```

Comment:
Yes but you have already the `../` after, and `./../` is equivalent to `../`.. :P